src/main/kotlin/org/opensearch/replication/task/shard/TranslogSequencer.kt

```diff
@@ -50,20 +57,30 @@ class TranslogSequencer(scope: CoroutineScope, private val replicationMetadata:
                         private val followerShardId: ShardId,
                         private val leaderAlias: String, private val leaderIndexName: String,
                         private val parentTaskId: TaskId, private val client: Client, initialSeqNo: Long,
-                        private val followerClusterStats: FollowerClusterStats) {
-
+                        private val followerClusterStats: FollowerClusterStats, writersPerShard : Int) {
+    //Start backOff for exceptions with a second
+    private val initialBackoffMillis = 1000L
```

Comment:
Seems to be redundant based on the logic from Extensions.kt, Could we re-use the method here?